osu.Game.Rulesets.Osu.Tests/TestSceneSlider.cs

```diff
@@ -187,6 +187,45 @@ public void TestChangeSamplesWithNodeSamples()
             bool assertSamples(HitObject hitObject) => hitObject.Samples.All(s => s.Name != HitSampleInfo.HIT_CLAP && s.Name != HitSampleInfo.HIT_WHISTLE);
         }
 
+        [Test]
+        public void TestSliderWithMultipleSegments()
+        {
+            AddStep("create slider", () =>
+            {
+                var bezierSegment = new List<Vector2> { Vector2.Zero };
+
+                for (int i = 1; i < 50; i++)
+                {
+                    bezierSegment.Add(new Vector2(i * 10, -300));
+                    bezierSegment.Add(new Vector2(i * 10, 300));
+                }
+
+                var slider = create(new Vector2(-200),
+                    new PathSegment(PathType.Bezier, new[]
+                    {
+                        Vector2.Zero,
+                        new Vector2(80, -125),
+                        new Vector2(150, 105),
+                        new Vector2(260, 0),
```

Comment:
shouldn't there be tests and exception logic if the previous segment's last vector doesn't equal the first of the next segment?

Reply:
Will add a test but an exception isn't necessary since it works just fine (just interpolates linearly between the segments).

Original:
<img width="960" alt="Screen Shot 2019-11-19 at 12 56 49 pm" src="https://user-images.githubusercontent.com/1329837/69115486-129b5f80-0acc-11ea-8378-a241080a181c.png">

Separated:
<img width="960" alt="Screen Shot 2019-11-19 at 12 55 32 pm" src="https://user-images.githubusercontent.com/1329837/69115442-e253c100-0acb-11ea-857c-c979955b275a.png">

Reply:
hmm i guess that's okay, seems a bit weird though. i guess it's hard to avoid duplication of the end / start vectors of each segment to ensure this can't happen in the first place?

just thinking forward mappers may use these kinds of edge cases to create broken game mechanics.

Reply:
Given that sliders are computed as linear interpolations between pieces, it's more expected functionality rather than an edge case. Wouldn't be hard to add a check to avoid duplication but I see it as unnecessary.